B2B-811 - Add b2b english terms conventions (#933)
shopware/docs: products/extensions/b2b-suite/concepts/basic-conventions.md

# Basic Conventions

## Codebase conventions

> This is a list of naming conventions the B2B Suite complies to:

Reviewer: ```suggestion
List of naming conventions the B2B Suite complies to:
```

| Storefront plugins                                                                                 | File names end with *.plugin.ts                                                     |
| Interfaces                                                                                         | File names start with `I`, e.g., `IAjaxPanelEvent.ts`                                |
| Snippets                                                                                           | The root snippet key is `b2b`                                                       |

> ## Entity naming convention

Reviewer: ```suggestion
## Entity naming conventions
```

> In order to make a more comprehensible B2B UI for English storefronts, some domain entities (i.e: Contact, Contingent Rule) do not have parallel naming conventions between the B2B GUI and the codebase:

Reviewer: ```suggestion
Certain domain entities, such as Contact, Contingent Rule, and others, have different naming conventions in the B2B UI and the codebase. To make things easier to understand, here is a comparison between these entities' B2B UI to their corresponding names in English Storefronts:
```

| English term (Display name)                        | B2B-Suite term (Entity name)                      |
|----------------------------------------------------|---------------------------------------------------|

> | Company Administrator                              | Debtor                                            |

Reviewer: ```suggestion
| Company administrator                              | Debtor                                            |
```
